### ISSUE TITLE
VGG model location

### ISSUE THREAD

**Issue by xslittlegrass** [dmlc/mxnet-model-gallery#20]
It seems that the URL for VGG model is not correct in the description.

The model is located at

http://www.mxnet.io/models/imagenet/vgg/

instead of 

http://data.dmlc.ml/mxnet/models/imagenet/vgg

**Comment by tornadomeet** [comment]
@xslittlegrass  yes, you are right, the later is the old server. would you fix this and give a pr? thanks.

**Comment by xslittlegrass** [comment]
@tornadomeet pr sent. Thanks.

**Comment by mheyman** [comment]
VGG seems to be misplaced again and I cannot locate it...

**Comment by msollami** [comment]
Yes it's missing.

**Comment by aseuteurideu** [comment]
may be other alternative will be download Caffe's VGG-16 model and then convert it by tools in tools/caffe_converter/convert_model.py
